docs/admin-guide/troubleshooting.md

```diff
@@ -93,114 +88,89 @@ The steps below are applicable for Sev 1 & 2 incidents, see the Escalations sect
   </tr>
 </table>
 
-
-
 ## Escalations
 
-After receiving a new alert or ticket, spend a few minutes (&lt; 5m) doing a preliminary investigation to identify the reproducibility and impact of the incident. Once confirmed as an issue being the formal incident response process:
-
-
-
-## Hypothesis development
+After receiving a new alert or ticket, spend a few minutes (&lt; 5m) do a preliminary investigation to identify the reproducibility and impact of the incident. Once confirmed as an issue, the formal incident response process is:
```

Comment:
My reading here is that the headings between "Escalations" and "Incident Resolution" are part of the "Escalations" subsection, but not sure.